src/ducks/HistoricalBalance/BalanceView.js

```diff
@@ -4,7 +4,7 @@ import GetHistoricalBalance from './GetHistoricalBalance'
 import HistoricalBalanceChart from './HistoricalBalanceChart'
 import AssetsField from './AssetsField'
 import styles from './BalanceView.module.scss'
-import SignalMasterModalForm from '../Signals/SignalModal/SignalMasterModalForm'
+import SignalMasterModalForm from '../Signals/signalModal/SignalMasterModalForm'
```

Comment:
Why react component folder name was changed to a camelCase?

Reply:
@Partysun  wants to rename folders in camelCase

Reply:
The conversation was about `Redux` which does not contain any react component